### PR TITLE
read from cached manifests if exist for mobvoi

### PR DESCRIPTION
**Pull request by freewym**
@pzelasko Do you think the function `read_if_cached` is general enough to be moved to utils.py?

### CODE REVIEW

**Comment by pzelasko** [comment]
I think maybe it could work with most of the corpora as-is. I'd like to eventually generalize some solutions from the librispeech recipe (especially this and the multi-threaded preparation) to work with other corpora. Feel free to move it, and if you have any ideas how to make it more generic, these are also welcome :)

**Comment by freewym** [comment]
@pzelasko Does the codecov check matter? Other than that, this PR is ready.

File: lhotse/recipes/utils.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from lhotse.supervision import SupervisionSet
+from lhotse.utils import Pathlike
+
+
+def read_manifests_if_cached(
+        dataset_parts: Optional[Sequence[str]],
+        output_dir: Optional[Pathlike]
+) -> Optional[Dict[str, Dict[str, Union[RecordingSet, SupervisionSet]]]]:
+    """Loads manifests from the disk if all of them exist in the specified paths.
+    This function is intended to speedup data preparation if it is already done before.
```
Comment (pzelasko) [inline lhotse/recipes/utils.py:15]: Could you mention in the doc that the manifests are searched for using the pattern `output_dir / f'{manifest}_{part}.json'`, where `manifest` is one of `["recordings", "supervisions"]`?

**Comment by pzelasko** [comment]
No need to worry about codecov for data prep scripts. I left one minor comment, and then LGTM.

**Comment by freewym** [comment]
Done

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**